arkouda/pdarrayclass.py

```diff
@@ -1168,6 +1188,38 @@ def info(pda : Union[pdarray, str]) -> str:
         raise TypeError("info: must be pdarray or string".format(pda))
         return generic_msg(cmd="info", args="{}".format(pda))
 
+def list_registry() -> list[str]:
+    """
+    Return a list containing the names of all registered objects
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    list
+        List of all object names in the registry
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there's a server-side error thrown
+    """
+    registered_list: list[str] = []
+
+    if info('__RegisteredSymbols__') == '__EMPTY_REGISTRY__':
+        return registered_list
```

Comment:
No change requested, just stylistic:
You could wrap the for loop below in this if condition (flipping to != comparison) and reduce it to a single exit point.
i.e.

````
registered_list: List[str] = []
if info(...) != '__EMPTY_REGISTRY__':
  for (...):
    # update registered_list as necessary
    registered-list.append(...)

return registered_list
```

Reply:
I think I'd rather have defined symbols like `ak.EmptyRegistry` and `ak.EmptySymbolTable` instead of string literals showing up in the code.

Reply:
Sounds good! I can make that change. I'll also update references to this in registration tests